### ISSUE TITLE
Visual bug in layout of channel header and date

### ISSUE THREAD

**Issue by venkatasg** [ZeeZide/Shrugs#2]
<img width="602" alt="Screen Shot 2020-06-04 at 10 38 03 PM" src="https://user-images.githubusercontent.com/22871413/83834499-0e274480-a6b4-11ea-9a7e-56a8946783b9.png">

The screenshot above should illustrate the issue I'm talking about. There is a thinspacing between the bar that has the channel purpose, and the translucent bar that has the send date of messages below. This isn't a bug, but just a visual annoyance.

**Comment by helje5** [comment]
It is actually intentional (a custom implementation to reach this effect) for two reasons:
- it doesn't look good if that bar sticks to the gray of the topic header
- to give a sense that there is "more above"

Maybe I should move it further down to avoid the impression that it is a glitch. Other suggestions welcome (different color but attach to the top?)

**Comment by venkatasg** [comment]
I understand that, which is why I was unsure whether to label this as a bug. I do think it looks a little ungainly with the white strip separation. Perhaps it may look better attached but without the transparency?

**Comment by nriley** [comment]
Here's a more pathological case with an image:

![image](https://user-images.githubusercontent.com/68303/83956247-cfb39600-a829-11ea-9562-0e662f636560.png)

The narrow strip of 100% opacity draws unnecessary attention to itself. I'm not sure you need to separate the topic and date bar as you can see content underneath the date bar in any case, so it is pretty clear it can be scrolled.  I can understand it might look weird to have two gray bars right next to one another; you could certainly choose another color!

What about combining the topic and the date into a single gray, translucent bar (topic left justified, date right justified as it is now)? That'd save vertical space too.

**Comment by helje5** [comment]
![Screenshot 2020-06-07 at 13 56 15](https://user-images.githubusercontent.com/7712892/83968049-a85ed680-a8c6-11ea-831f-7589f31eda56.png)

The thing is, those are floating tableview grouping headers, they are not sticky the way you may think.

@martinlexow, any suggestions on how to improve those? 😬

**Comment by nriley** [comment]
Yeah, I was only talking about the topmost one in that case. Not sure if this is possible with your current design though!

**Comment by martinlexow** [comment]
![barEffect](https://user-images.githubusercontent.com/42469550/84009310-5e7ffa00-a973-11ea-95b8-7c694640ee03.gif)
Here’s an example how I solved this in one of my apps. I think using a VisualEffect view as background combined with a separator works better: Good contrast and you still get a feel for the content behind it. To improve readability you could also think about putting the date on the left: It’s a bit far away from the rest at the moment. Travelling long distances with the eye repeatedly is tiring.
